install_config/persistent_storage/persistent_storage_glusterfs.adoc

```diff
@@ -34,6 +67,65 @@ High-availability of storage in the infrastructure is left to the underlying
 storage provider.
 ====
 
+[[gfs-support-requirements]]
+== Support Requirements
+
+The following requirements must be met to create a supported integration of Red
+Hat Gluster Storage and {product-title}.
+
+[[gfs-supported-operating-systems]]
+=== Supported Operating Systems
+
+The following table lists the supported versions of {product-title} with Red Hat
+Gluster Storage Server.
+
+[options="header"]
+|===
+|Red Hat Gluster Storage | {product-title}
+
+|3.1.3
+|3.1 or later
```

Comment:
I think this could be version 3.2 (no later or earlier as it's the only version we have validated against)